### ISSUE TITLE
add Corpus Coranicum TUK ids?

### ISSUE THREAD

**Issue by thea-m** [BetaMasaheft/Documentation#629]
Some Ethiopic texts have received a TUK (Umwelttext) id from Corpus Coranicum. These are mostly single verses or paragraphs, the complete list seems to be found here:
http://corpuscoranicum.de/kontexte/uebersicht
(Under "Altäthiopisch")
For example:
http://corpuscoranicum.de/kontexte/index/sure/1/vers/2/intertext/906
I think it would be useful to add these IDs to our records, even if they only refer to part of a work. What do you think?

**Comment by thea-m** [comment]
I think that it would make sense to add this information either by entering the IDs of these texts as a clavis ID or by having a relation (which would maybe be more appropriate in the cases in which the TUK only encompasses a part of one of our works?). What do you think, @PietroLiuzzo ?

**Comment by thea-m** [comment]
@PietroLiuzzo ?

**Comment by PietroLiuzzo** [comment]
Thanks for pinging me, 100% sure we want to link these. that the id 906 refers to is actually only a passage. I would then formulate the link from the passage to the passage.
<relation name="skos:exactMatch" 
active="http://corpuscoranicum.de/kontexte/index/sure/1/vers/2/intertext/906"
passive="urn:dts:betmas:LIT1340EnochE:1.3-5"
/>

However, it does not seam to me that we are using the same citation structure. I cannot see that text at the point it should be. our urn needs to point to our text of course, so if we have what they say is 1.3-5 under another numeration, unless all our text is falsely organized, then we need to put our passage.

**Comment by PietroLiuzzo** [comment]
I will have then to make sure that this is
- transferred to the RDF correctly
- that the id can be used as a URI

**Comment by thea-m** [comment]
thank you, I will have a look at it

**Comment by PietroLiuzzo** [comment]
if it works we should document this in the GL

**Comment by thea-m** [comment]
Regarding the citation structure of the example from the book of Enoch: We use the same text as reference, but in our records we have the division into books and chapters, while they use the chapters (not the books) and also follow the verse numbering given in the translation (and in the edition on the top of the pages, but not within the text, which is a photographed manuscript). So their 1.3-5 is indeed found within our book 1, chapter 1 div, but we cannot give a more exact reference to the verses. Should I  nevertheless add the relation to chapter 1?
And just for my understanding: Why is active here the external reference and passive our record?

**Comment by PietroLiuzzo** [comment]
Fro my point of view this is a meaningful link from their collection of passages to our full text, not viceversa. The pointer can then only point to the div and should be a broadMatch, not exactMatch, IMHO.

**Comment by thea-m** [comment]
it would not be an exact match then, is it broadMatch?

**Comment by thea-m** [comment]
makes sense, thank you!

**Comment by thea-m** [comment]
for works whose text we don't have, should the relations be added in the general listRelation in div bibliography?
```
<relation name="skos:broadMatch" 
active="https://corpuscoranicum.de/kontexte/index/sure/3/vers/146/intertext/929"
passive="LIT1697Jubilees"/>
```
and this be repeated for all intertexts relating to this work?

**Comment by PietroLiuzzo** [comment]
`listRelation` needs not to be in `div[@type='bibliography']`.
yes, the relation in this way looks fine to me, we will be able to update it in the future if we happen to get that text. schade that there is no skos:VERYbroadMatch ...

**Comment by thea-m** [comment]
yes, but that's where it is in that file, it's not necessary to move it or to open a new listRelation?

**Comment by PietroLiuzzo** [comment]
in the commit above, the app will now redirect correctly encoded uris like the one which will be in the relation table for urn
http://betamasaheft.eu/urn:dts:betmas:LIT1340EnochE:1.3
will redirect in 3.2 to
http://betamasaheft.eu/works/LIT1340EnochE/text?start=1
in case no header is set. if the json-LD accept request header is set, it will redirect to the DTS collection API, i.e.
http://betamasaheft.eu/api/dts/collections?id=urn:dts:betmas:LIT1340EnochE

**Comment by thea-m** [comment]
for my part, this can be closed, @PietroLiuzzo ?
